src/esp/gfx/MaterialData.h

```diff
@@ -25,9 +25,21 @@ struct PhongMaterialData : public MaterialData {
   Magnum::GL::Texture2D *ambientTexture = nullptr, *diffuseTexture = nullptr,
                         *specularTexture = nullptr, *normalTexture = nullptr;
   bool perVertexObjectId = false, vertexColored = false;
+  std::string importName;
 
   ESP_SMART_POINTERS(PhongMaterialData)
 };
 
+struct PythonMaterial {
```

Comment:
Instead of making a port for python, let's have the phong material data have two parts, the material assets (the textures) and the material parameters (these things).  Material assets don't have a reason to go to python but then the material parameters can.